gtests/net/mptcp/mp_join/mp_join_server_v4.pkt

```diff
@@ -16,8 +16,14 @@
 +0.0 < . 1:1(0) ack 1 win 256 <nop,nop,TS val 4074410674 ecr 4074410674,mpcapable v1 flags[flag_h] key[ckey=2,skey]>
 +0    accept(3, ..., ...) = 4
 
-+1.0 < P. 1:3(2) ack 1 win 256 <nop,nop,TS val 4074418292 ecr 4074410674,mpcapable v1 flags[flag_h] key[skey,ckey] mpcdatalen 2,nop,nop>
-+0.0 > . 1:1(0) ack 3 <nop,nop,TS val 4074418293 ecr 4074418292,add_address addr[saddr1] hmac=auto,dss dack8=3 dll=0 nocs>
+// add_addr
++0.0 > . 1:1(0) ack 1 <nop,nop,TS val 4074418293 ecr 4074418292,add_address addr[saddr1] hmac=auto,dss dack4=1 dll=0 nocs>
+// TODO: send echo bit
+// +0.0 > . 1:1(0) ack 1 win 256 <nop,nop,TS val 4074418293 ecr 4074418292,add_address addr[saddr1],dss dack8=1 dll=0 nocs>
+
++0.2 < P. 1:3(2) ack 1 win 256 <nop,nop,TS val 4074418292 ecr 4074410674,mpcapable v1 flags[flag_h] key[skey,ckey] mpcdatalen 2,nop,nop>
+// TODO: Why dack8 here?
++0.0 > . 1:1(0) ack 3 <nop,nop,TS val 4074418293 ecr 4074418292,dss dack8=3 dll=0 nocs>
```

Comment:
is it normal our kernel send dack8 here? We didn't receive any DSS in 64 bits so far, no?

Reply:
A packet trace for this test: [dack8.zip](https://github.com/multipath-tcp/packetdrill/files/5860041/dack8.zip)

Reply:
That's the expected behaviour: https://github.com/multipath-tcp/mptcp_net-next/blob/export/20201119T060730/net/mptcp/options.c#L997

Not knowing if the sender wants to use 64-bit or 32-bit DSN, we reply with 64-bit ACK to be "on the safe side". We will adapt that later if needed.